packages/angular-test/src/text.ts

```diff
@@ -250,7 +254,7 @@ export const textTest = <C extends JsonFormsControl>(
             mockSubStore.complete();
             fixture.detectChanges();
             component.ngOnInit();
-            expect(textNativeElement.id).toBe('myId');
+            expect(textElement.nativeElement.id).toBe('myId');
```

Comment:
Shouldn't we use `textNativeElement` here?

Reply:
no as textNativeElement could be in a wrapper and we can't be sure that the propagation is working as expected, eg for angular-material your suggestions works, but for ionic it fails

Reply:
We shoud improve naming then (maybe via #1100), I'd think that both `textNativeElement` and `textElement.nativeElement` are likely to be used as synonymous.